tests/github/interactive_api_director/experiments/pytorch_kvasir_unet/envoy/shard_config.yaml

```diff
@@ -0,0 +1,5 @@
+template: kvasir_shard_descriptor.KvasirShardDescriptor
+params:
+  data_folder: kvasir_data
+  rank_worldsize: 1,90
+  enforce_image_hw: '300,400'
```

Comment:
This object reflects 'size' type that by design consists of fixed width and height. For example, the same approach is used for `Resize*` transforms in [`torchvision.transforms`](https://pytorch.org/vision/stable/transforms.html), where `size` variable is passed as a tuple or single integer, which means that `h=w`. In my opinion, these values should be stored together since they are always used together.

Reply:
`rank_worldsize` and `enforce_image_hw` have diferent types `list` and `string`. It looks like it should be more consistent